.github/workflows/build.yaml

```diff
@@ -115,6 +115,22 @@ jobs:
             make push-${{ matrix.image-name }}; \
           done;
 
+    - name: Pull cache, build and push dev image
+      if: ${{ matrix.image-name == 'api' }}
```

Comment:
We have a mechanism for skipping images, let's add an option to skip `api-dev` as well.